mantaray/received.py

```diff
@@ -386,6 +431,19 @@ def _handle_endofmotd(server_view: views.ServerView) -> None:
         server_view.core.send(f"JOIN {channel}")
 
 
+def _handle_whoreply(
+    server_view: views.ServerView, command: str, args: list[str]
+) -> None:
+    assert len(args) == 8
+    nick = args[5]
+    away_status = args[6][0]
+    view = server_view.find_channel(args[1])
+
+    assert view is not None
+    if away_status.lower() == "g":
```

Comment:
I would assert that it's either `"g"` or `"h"`. If it's something else, it will currently assume the user isn't away, which could be confusing to debug.

Reply:
It should be impossible that it's something else, but I can add it for sanity.